Spec/stargaze.json

```diff
@@ -0,0 +1,798 @@
+{
+    "proposal": {
+        "title": "Add Specs: Stargaze",
+        "description": "Adding new specification support for relaying agoric data on stargaze",
+        "specs": [
+            {
+                "index": "STRGZ",
+                "name": "stargaze mainnet",
+                "enabled": true,
+                "imports": [
+                    "COSMOSSDKFULL"
```

Comment:
needs to inherit also from cosmossdkv50

Reply:
https://github.com/lavanet/lava-config/pull/13/commits/23267b06fa9e976fda40c27cbeeb02420b6c3bcc

Fixed here